### PR TITLE
Adding examples from example-bundles repo

### PR DESCRIPTION
**Pull request by jeremyrickard**
This is a WIP PR that will add relevant examples from the example-bundles repo. 

Signed-off-by: Jeremy Rickard <jeremy.rickard@microsoft.com>

### CODE REVIEW

File: examples/wordpress-mysql/bundle.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
+{
+    "name": "wordpress-mysql",
+    "version": "0.2.0",
+    "schemaVersion": "v1.0.0-WD",
```
Comment (radu-matei) [inline examples/wordpress-mysql/bundle.json]: This should be `v1.0.0`

Comment (radu-matei) [reply examples/wordpress-mysql/bundle.json]: Also, could you please update `duffle.json` from the `helloworld` example as well to the latest schema version?

File: examples/aks-terraform/cnab/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    apk update && \
+    apk add bash py-pip && \
+    apk add --virtual=build gcc libffi-dev musl-dev openssl-dev python-dev make && \
+    pip install --upgrade pip && \
+    pip install azure-cli
+
+COPY Dockerfile /cnab/Dockerfile
+COPY app /cnab/app
+
+RUN chmod 755 /cnab/app/run
```
Comment (radu-matei) [inline examples/aks-terraform/cnab/Dockerfile]: Tried to do a `duffle build` on this directory:

```
Step 5/9 : COPY Dockerfile /cnab/Dockerfile
 ---> abb6b13c59c9
Step 6/9 : COPY app /cnab/app
 ---> 6621ce9616c0
Step 7/9 : RUN chmod 755 /cnab/app/run
 ---> Running in 447135ad9430
chmod: /cnab/app/run: No such file or directory
Error: error building image: error building image cnab: error streaming messages for image builder cnab with builder docker: The command '/bin/sh -c chmod 755 /cnab/app/run' returned a non-zero code: 1
```

Comment (jeremyrickard) [reply examples/aks-terraform/cnab/Dockerfile]: oops, missed this and the init-backend when i added files, my bad!

**Comment by vdice** [comment]
@jeremyrickard is this PR ready for review?  Eager to get this in and archive the `deislabs/example-bundles` if we can.

**Comment by radu-matei** [comment]
I think this also needs a rebase.

**Comment by vdice** [comment]
Closing in favor of https://github.com/cnabio/duffle/pull/888.  Thanks for the huge headstart @jeremyrickard !

**Pull request closed without merging**